src/ai/backend/manager/api/schema.graphql

```diff
@@ -609,20 +609,20 @@ type UserResourcePolicy {
   name: String!
   created_at: DateTime!
 
-  """Added since 24.03.1. Limitation of the number of user vfolders."""
+  """Added in 24.03.1. Limitation of the number of user vfolders."""
```

Comment:
According to the PR description, `max_vfolder_count` field of {User/Project}ResourcePolicy is added in "23.09.6". But here is 24.03.1. Which one is correct?

Reply:
`max_vfolder_count` field was added in 23.09.6 in 23.09 major version, and it was added in 24.03.1 in 24.03 major version.
I'm sorry that I didn't update the PR description.